json-checks/src/main/java/org/sonar/json/checks/CheckList.java

```diff
@@ -36,6 +36,7 @@ private CheckList() {
 
   @SuppressWarnings("rawtypes")
   public static Collection<Class> getChecks() {
+    System.out.println("kkkk");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)